File: src/CSharp.Forms.Data/DataContext.cs
using CSharp.Forms.Data.Models;
using Microsoft.EntityFrameworkCore;
using Microsoft.Extensions.Configuration;

namespace CSharp.Forms.Data
{
    public class DataContext : DbContext
    {
        protected readonly IConfiguration Configuration;

        public DataContext(IConfiguration configuration)
        {
            Configuration = configuration;
        }

        protected override void OnConfiguring(DbContextOptionsBuilder options)
        {
            // connect to sqlite database
            options.UseInMemoryDatabase(Configuration.GetConnectionString("WinformsDatabase"));
        }

        public DbSet<User> Users { get; set; }

    }
}

File: VB.Forms.Data/DataContext.vb

Imports Microsoft.EntityFrameworkCore
Imports Microsoft.Extensions.Configuration

Namespace VB.Forms.Data

    Public Class DataContext
        Inherits DbContext
        Protected ReadOnly Configuration As IConfiguration

        Public Sub New(ByVal configuration As IConfiguration)
            Me.Configuration = configuration
        End Sub

        Protected Overrides Sub OnConfiguring(ByVal options As DbContextOptionsBuilder)
            ' connect to sqlite database
            options.UseInMemoryDatabase(Configuration.GetConnectionString("WinformsDatabase"))
        End Sub

        Public Property Users As DbSet(Of User)

    End Class

End Namespace
